### ISSUE TITLE
Making mailpit relay work with Outlook

### ISSUE THREAD

**Issue by fr6nco** [axllent/mailpit#118]
Hi,

Im trying to make mailpit relay work with outlook's smtp mail server, with the following configuration

```
      host: smtp-mail.outlook.com
      port: 587
      starttls: true
      allow-insecure: true
      auth: plain
      username: [email-here]
      password: [password-here]
```

with no luck tho.

Im getting this error

```
WARN[2023/05/29 21:34:47] [smtp] error relaying message: 504 5.7.4 Unrecognized authentication type [DUZPR01CA0003.eurprd01.prod.exchangelabs.com 2023-05-29T21:34:42.410Z 08DB5FD917A96013]
```

According to https://support.microsoft.com/en-us/office/pop-imap-and-smtp-settings-for-outlook-com-d088b986-291d-42b8-9564-9c414e2aa040 configuration seems to be correct.

**Comment by axllent** [comment]
@fr6nco Apparently Office365 does not support any of the existing Mailpit SMTP login methods (eg: PLAIN), and requires either LOGIN (username/password) or XOAUTH2 (2-factor authentication). XOAUTH2 is not an option, so assuming you can send using a username and password, I can try add the LOGIN method to Mailpit.

The issue here is I have no means of testing it first. Can you please tell me what OS you are testing on, then maybe I can provide you with a beta version so you can verify whether LOGIN works before I add it to the main code?

**Comment by fr6nco** [comment]
Im running on kubernetes on x86 with your docker image.
```
image:
  registry: docker.io
  repository: axllent/mailpit
  tag: v1.6.10
```

If you could push a dev tag, that would be perfect.
Thanks for the blazingly fast response

**Comment by fr6nco** [comment]
I built the docker image and deployed it. Email went through with no issues

```
DEBU[2023/05/30 08:17:06] [smtp] relayed message from az-support@mad-me.com via smtp-mail.outlook.com:587
```

If u want to test it yourself, here's a public image available https://hub.docker.com/r/madmedocker/mailpit/tags

**Comment by axllent** [comment]
This has been released as v1.6.13. Please feel free to close this if it works as expected, thank you.

**Comment by fr6nco** [comment]
Perfect, tested. works well.

Here's my merge request to the helm chart in case anyone would like to deploy mailpit with SMTP relay config.
https://github.com/jouve/charts/pull/1
